example/jupyter/tutorial_xgboost.ipynb

```diff
@@ -0,0 +1,357 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# XGBoost on SQLFlow Tutorial\n",
+    "\n",
+    "This is a tutorial on train/predict XGBoost model in SQLFLow, you can find the design doc from [here](https://github.com/sql-machine-learning/sqlflow/blob/develop/doc/xgboost_on_sqlflow_design.md), in this tutorial you will know how to:\n",
+    "- Train a XGBoost model to fit the boston housing price; and\n",
+    "- Predict the housing price using the trained model;\n",
+    "\n",
+    "\n",
+    "## The Dataset\n",
+    "\n",
+    "This tutorial would use the [Boston Housing](https://www.kaggle.com/c/boston-housing) as the demonstration dataset.\n",
+    "The database contains 506 lines and 14 columns, the meaning of each column is as follows:\n",
+    "\n",
+    "Column | Explain \n",
+    "-- | -- \n",
+    "crim|per capita crime rate by town.\n",
+    "zn|proportion of residential land zoned for lots over 25,000 sq.ft.\n",
+    "indus|proportion of non-retail business acres per town.\n",
+    "chas|Charles River dummy variable (= 1 if tract bounds river; 0 otherwise).\n",
+    "nox|nitrogen oxides concentration (parts per 10 million).\n",
+    "rm|average number of rooms per dwelling.\n",
+    "age|proportion of owner-occupied units built prior to 1940.\n",
+    "dis|weighted mean of distances to five Boston employment centres.\n",
+    "rad|index of accessibility to radial highways.\n",
+    "tax|full-value property-tax rate per \\$10,000.\n",
+    "ptratio|pupil-teacher ratio by town.\n",
+    "black|1000(Bk - 0.63)^2 where Bk is the proportion of blacks by town.\n",
+    "lstat|lower status of the population (percent).\n",
+    "medv|median value of owner-occupied homes in $1000s.\n",
+    "\n",
+    "We seperated the dataset into train/test datasets which used to train/predict our model."
```

Comment:
SQLFlow will auto split train/validation dataset, shall we mention this?

Reply:
Good idea, fixed.